lib/cloudregister/registerutils.py

```diff
@@ -173,6 +175,96 @@ def clear_rmt_as_scc_proxy_flag():
         pass
 
 
+# ----------------------------------------------------------------------------
+def clean_payg_extensions():
```

Comment:
While the extensions we are removing are paid extensions they are not PAYG technically, using the `payg` moniker here can be confusing to the reader, I think we should stick with the `non_free` term, i.e. `clean_non_free_extensions`